lib/librato/rack.rb

```diff
@@ -37,6 +37,32 @@ module Librato
   #   run MyApp
   #
   class Rack
+    RECORD_RACK_IMPL = <<-'EOS'
+      if tracker.suite_enabled?(:rack)
```

Comment:
I think this `if` conditional could be removed because this only gets inserted if `rack` is enabled, right?